Editor/Resources/EditorWindow/Components/ConnectToFleetInput.uxml

```diff
@@ -0,0 +1,40 @@
+﻿<!-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<ui:UXML xmlns:ui="UnityEngine.UIElements">
+    <ui:Style src="./common.uss"/>
+    <ui:VisualElement name="AnywherePageCreateFleet" class="separator">
+        <ui:VisualElement name="AnywherePageCreateFleetName" class="form-row">
+            <ui:Label name="AnywherePageCreateFleetNameLabel" class="form-row__label" text="Fleet Name"/>
+            <ui:VisualElement class="form-row__input">
+                <ui:TextField name="AnywherePageCreateFleetNameInput" class="text-input" value="GameName-AnywhereFleet"/>
+                <ui:Label name="AnywherePageCreateFleetNameHint" class="input-hint" text="Fleet Name must have 1–1024 characters. Valid characters are A-Z, a-z, 0-9, _ and - (hyphen)"/>
+            </ui:VisualElement>
+        </ui:VisualElement>
+        <ui:VisualElement class="form-row">
+            <ui:VisualElement class="form-row__input form-row__input--no-label separator separator--horizontal">
+                <ui:Button name="AnywherePageCreateFleetButton" class="button button--primary" text="Create Anywhere Fleet"/>
+                <ui:Button name="AnywherePageCreateFleetCancelButton" class="button" text="Cancel"/>
+            </ui:VisualElement>
+        </ui:VisualElement>
+    </ui:VisualElement>
+    <ui:VisualElement name="AnywherePageConnectFleet" class="separator">
+        <ui:VisualElement name="AnywherePageConnectFleetName" class="form-row">
+            <ui:Label name="AnywherePageConnectFleetNameLabel" class="form-row__label" text="Fleet Name"/>
+            <ui:DropdownField name="AnywherePageConnectFleetNameDropdown" class="form-row__input dropdown"/>
+        </ui:VisualElement>
+        <ui:VisualElement name="AnywherePageConnectFleetID" class="form-row">
+            <ui:Label name="AnywherePageConnectFleetIDLabel" class="form-row__label" text="Fleet ID"/>
+            <ui:Label name="AnywherePageConnectFleetIDDisplay" class="form-row__text-display" text="1234567890"/>
```

Comment:
See comment about renaming `form-row__text-display` https://github.com/aws/amazon-gamelift-plugin-unity/pull/96#discussion_r1335229666

Reply:
`form-row__text-display` is used for any field that is in the "input column" and used to position the text when it is not an input. `input-indicator` doesn't make sense in this context.

Reply:
I see, when I saw this in 96, I only noticed the status indicators being flagged with this class. With the follow-up PR (https://github.com/aws/amazon-gamelift-plugin-unity/pull/114) we're removing this class from the status indicators so moving on